### PR TITLE
Graylog adaptation

### PR DESCRIPTION
**Pull request by Meat-Chopper**

### CODE REVIEW

**Comment by trusche** [comment]
Hiya @Meat-Chopper, this is great, thanks for the PR. If you could add some tests that cover the new feature, I'd be happy to merge this.

**Comment by Meat-Chopper** [comment]
Hello @trusche, the tests have been added

File: spec/lib/http_client_spec.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 describe HTTPClient do
   let(:client) { HTTPClient.new }
+  before { HttpLog.configure { |c| c.logger = Logger.new @log } }
```
Comment (trusche) [inline spec/lib/http_client_spec.rb]: Why is this necessary here, isn't that the default already?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -43,3 +40,19 @@ def log
     @log.string
   end
 end
+
+class GelfMock < Logger
```
Comment (trusche) [inline spec/spec_helper.rb]: `spec_helper` doesn't seem the right place to define this - can we move it to a dedicated file somewhere?

File: spec/lib/http_client_spec.rb (hunk 1 above)
Comment (Meat-Chopper) [reply spec/lib/http_client_spec.rb]: I removed this default behavior from spec_helper.rb to be able to set another logger in tests

File: spec/spec_helper.rb (hunk 2 above)
Comment (Meat-Chopper) [reply spec/spec_helper.rb]: Applied

File: spec/lib/http_client_spec.rb (hunk 1 above)
Comment (trusche) [reply spec/lib/http_client_spec.rb]: Yep, I get that, but this is the default which is now set in `spec/lib/http_log_spec.rb` - I don't understand the need to set it explicitly again here, just for HttpClient - does it actually change anything from the default?

Comment (Meat-Chopper) [reply spec/lib/http_client_spec.rb]: I did that because HTTPClient test had failed :) It looks like spec/lib/http_log_spec.rb and spec/lib/http_client_spec.rb are independent files. And before actions from one file are not applied to another.

**Comment by Meat-Chopper** [comment]
Thank you!

**Comment by trusche** [comment]
Published as v1.3.3. Thanks!
